integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/ClusterTemplateTest.java

```diff
@@ -49,6 +50,8 @@
 
     private static final String VALID_LOCATION = "London";
 
+    private static final String VALID_CT_NAME = "ClusterTemplate";
```

Comment:
The cleanup mechanism will not find the cluster template resource and it will be stuck.